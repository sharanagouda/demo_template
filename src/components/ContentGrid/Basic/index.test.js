import { Basic } from ".";

const data = {
  attributes: { heading: "" },
  totalCount: 100,
  items: [
    {
      route: "goldrausch-in-alaska/1000-unzen",
      name: "1011 Unzen",
      secondaryTitle: "",
      image:
        "https://eu1-test-images.disco-api.com/2017/11/19/videoasset-16852-1511063653344.png?f=jpeg&p=true&q=75&w=316"
    }
  ]
};

test("render correct component markup", () => {
  const container = new Basic(data).render();

  expect(container.element).toMatchSnapshot();
});

test("contains scrollManager", () => {
  const container = new Basic(data);

  expect(container.scrollManager).toBeDefined();
});

test("contains items", () => {
  const container = new Basic(data);

  expect(container.items).toHaveLength(1);
});
