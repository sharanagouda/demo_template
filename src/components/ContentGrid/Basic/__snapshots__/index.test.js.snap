// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correct component markup 1`] = `
<div
  class="container"
>
  <div
    cdtv_sn_overflow_ignore="true"
    class="scroll-manager container_horizontal"
  >
    <div
      class="outer"
    >
      <div
        class="inner"
      />
    </div>
  </div>
</div>
`;
